pages/blog/_posts/2021-02-28-kafka-timers.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: blogpost
+permalink: /blog/kafka-timers
+title: "Kafka timers"
+date: 2021-02-28
+tags: kafka timer stateless
+author: <a href="https://www.linkedin.com/in/sergey-zyrianov-8a7ab34/">Sergey Zyrianov</a> - Tech Lead
+---
+
+A timer is a cornerstone of any software that communicates over the network. There are plenty of implementations
+that provide a timer facility. Most of them are in memory and will lose scheduled timers should the application crash.
+In this blog we discuss durable Kafka timers that do not depend on in memory state. By design
+we shall assume 1 second resolution of these timers.
+---
+
+Before taking on timers let's cover some Kafka basics.
+
+# Kafka concepts
```

Comment:
The `title` property from the frontmatter (`title: "Kafka timers"`) is `#` (h1), so all your headings should have one extra `#`.

```
## Kafka concepts
## Implementation
### Oscillator
```

etc